app/utils/whitelabel.ts

```diff
@@ -1,31 +1,89 @@
+import AskDarcelImage from '../assets/img/askdarcel.svg';
+import BackgroundImage from '../assets/img/bg.png';
+import SearchByAlgoliaImage from '../assets/img/search-by-algolia.png';
+import SFFamiliesLogo from '../assets/img/sf-families.svg';
+import SFServiceLogo from '../assets/img/sf-service.svg';
+import SFSeal from '../assets/img/sf-seal.png';
 import config from '../config';
+import icon from '../assets';
+import styles from '../components/ui/Navigation/Navigation.module.scss';
 
-/** Whether we should display the SF Service Guide branded version of the site */
-const isSFFamiliesSite = () => window.location.host.indexOf(config.SFFAMILIES_DOMAIN) > -1;
+// Include new white label here
+type WhiteLabelSiteKey = 'default' | 'SFServiceGuide' | 'SFFamilies';
 
-const isSFServiceGuideSite = () => !isSFFamiliesSite()
-  && window.location.host.indexOf(config.MOHCD_DOMAIN) > -1;
+interface WhiteLabelSite {
+  title: string;
+  siteUrl: string;
+  appImages: {
+    background: string;
+     // For appImages, logoLarge and logoSmall are usually different between each white label
+    logoLarge: string;
+    logoSmall: string;
+    algolia: string;
+    mohcdSeal: string;
+    icon: any;
+  },
+  styles?: any
+}
 
-const getSiteTitle = () => {
-  if (isSFServiceGuideSite()) {
-    return 'SF Service Guide';
-  }
-  if (isSFFamiliesSite()) {
-    return 'SF Families';
-  }
-  return 'AskDarcel';
+// Read only to force developer to modify configurations here, disallow changes at compile time
+const configurations: Partial<Record<Readonly<WhiteLabelSiteKey>, Readonly<WhiteLabelSite>>> = {};
```

Comment:
I want to make sure that this is the central place to modify whitelabel data so thus I made this read only.

Reply:
You don't need to mark `WhiteLabelSiteKey` as `Readonly`, since they're strings, and strings are immutable in JavaScript.

It also might be cleaner to put this at the bottom of the file so that you don't have the `Partial<>` anymore, and so that you can just mark the whole thing as readonly. For example, if you did:

```ts
const SFFamilies = {
  title: 'SF Families',
  // etc
} as const;
const SFServiceGuide = { /* stuff goes here */ } as const;
const default = { /* stuff goes here */ } as const;

const configurations: Readonly<Record<WhiteLabelSiteKey, WhiteLabelSite>> = {
  SFFamilies,
  SFServiceGuide,
  default,
} as const;
```